docs/update_and_migration/migrate_to_ibexa_dxp/common_issues.md

```diff
@@ -44,41 +45,41 @@ php bin/console ezplatform:images:normalize-path
 
 ## Unknown relation type 0
 
-"Unknown relation type 0." error occurs only when using REST API. The issue does not occur
+"Unknown relation type 0." error occurs only when using REST API. The issue doesn't occur
 the first time article is published (upon creation). It only happens after the article is
 edited and published.
 
-If this error occurs use the console command below. It will clean up redundant Relations rows:
+If this error occurs use the console command below. It cleans up redundant Relations rows:
 
 ```
 php bin/console ezpublish:update:legacy_storage_clean_up_relation_type_eq_zero
 ```
 The command can be executed in two modes:
 
-- list / dry-run - prints table with all corrupted Relations that will be deleted (to be executed first)
+- list / dry-run - prints table with all corrupted Relations that are deleted (to be executed first)
 - fix - executes clean up
 
 You can read more about this issue here: [EZP-27254](https://issues.ibexa.co/browse/EZP-27254)
 
-## Always available flag set on all Fields
+## Always available flag set on all fields
 
-Always available flag is set on all Fields, instead of only on Fields in the main
+Always available flag is set on all fields, instead of only on fields in the main
 language. This problem occurs when [[= product_name =]] is used to create content that is both
-always available and has multiple translations. The cleanup script will correctly set
+always available and has multiple translations. The cleanup script correctly sets
 always available flag for prioritized language filtering in Legacy search engine.
 
 ```
 php bin/console ezpublish:update:legacy_storage_fix_fields_always_available_flag
 ```
 
-Only affected Fields will be processed by the cleanup command.
+Only affected fields are processed by the cleanup command.
 
 You can read more about this issue here: [EZP-24882](https://issues.ibexa.co/browse/EZP-24882)
 
 ## Listing sub-content
 
-It is possible that after upgrade `sort_key_string` is left empty. This may cause problems
-in searches throughout the API. The cleanup script will check if Fields of given Field
+it's possible that after upgrade `sort_key_string` is left empty. This may cause problems
+in searches throughout the API. The cleanup script checks if fields of given field
 Type have correct sort key, and update it if needed.
```

Comment:
```suggestion
It's possible that after upgrade `sort_key_string` is left empty.
This may cause problems in searches throughout the API.
The cleanup script checks if fields of given field type have correct sort key, and update it if needed.
```